nvm.sh

```diff
@@ -704,6 +712,98 @@ nvm_binary_available() {
   nvm_version_greater_than_or_equal_to "$(nvm_strip_iojs_prefix "${1-}")" v0.8.6
 }
 
+nvm_set_colors() {
+  if [ "${#1}" -eq 5 ] && nvm_echo "$1" | nvm_grep -E "^[rRgGbBcCyYmMkKeW]{1,}$" 1>/dev/null; then
+    if nvm_has_colors; then
+      local INSTALLED_COLOR
+      local LTS_AND_SYSTEM_COLOR
+      local CURRENT_COLOR
+      local NOT_INSTALLED_COLOR
+      local DEFAULT_COLOR
+
+      INSTALLED_COLOR="$(echo "$1" | awk '{ print substr($0, 1, 1); }')"
+      LTS_AND_SYSTEM_COLOR="$(echo "$1" | awk '{ print substr($0, 2, 1); }')"
+      CURRENT_COLOR="$(echo "$1" | awk '{ print substr($0, 3, 1); }')"
+      NOT_INSTALLED_COLOR="$(echo "$1" | awk '{ print substr($0, 4, 1); }')"
+      DEFAULT_COLOR="$(echo "$1" | awk '{ print substr($0, 5, 1); }')"
+      nvm_echo_with_colors "Setting colors to: \033[$(nvm_print_color_code "${INSTALLED_COLOR}") ${INSTALLED_COLOR}\033[$(nvm_print_color_code "${LTS_AND_SYSTEM_COLOR}") ${LTS_AND_SYSTEM_COLOR}\033[$(nvm_print_color_code "${CURRENT_COLOR}") ${CURRENT_COLOR}\033[$(nvm_print_color_code "${NOT_INSTALLED_COLOR}") ${NOT_INSTALLED_COLOR}\033[$(nvm_print_color_code "${DEFAULT_COLOR}") ${DEFAULT_COLOR}\033[0m"
+
+      export NVM_COLORS="$1"
+    fi
+  elif nvm_has_colors; then
+    nvm help
+    nvm_echo
+    nvm_err_with_colors "\033[1;37mPlease pass in five \033[1;31mvalid color codes\033[1;37m. Choose from: rRgGbBcCyYmMkKeW\033[0m"
+    return 1
+  else
+    nvm help --no-colors
```

Comment:
i think the help output shouldn't come from this function, but only from the top-level `set-colors` code.

Reply:
Thanks, this change is now made.